task/ab-load/0.1/README.md

```diff
@@ -0,0 +1,82 @@
+# AB-load
+
+The ab-load task execute an http/https load test again't a targeted endpoint using the Apache benchmark toolkit.
+
+## Install the AB-load task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/startxfr/tekton-catalog/main/task/ab-load/0.1/ab-load.yaml
+```
+
+## Parameters
+
+| Key              | Default                              | Description                                                                                         |
+| ---------------- | ------------------------------------ | --------------------------------------------------------------------------------------------------- |
+| url              | `http://myapp.target_cluster.local/` | The URL targeted by the load test (must end with / if root domain is targeted)                      |
+| method           | `GET`                                | The HTTP method name                                                                                |
+| type             | `application/x-www-form-urlencoded`  | The http type of payload sended to the target endpoint (for POST and PUT method)                    |
+| size             | `1000`                               | Number of requests to perform                                                                       |
+| concurrency      | `10`                                 | Number of concurent requests to execute                                                             |
+| timeout          | `5`                                  | Seconds to wait for each response before ending up the connection and considering it's a bad answer |
+| additionnalFlags | `-k -I`                              | All ab command supported flag (ex: -k for keepalive or -I for insecure https connection)            |
+
+## Workspaces
+
+| Key          | Mount         | Description                                                                                                                                                                                                   |
+| ------------ | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| body-content | `/root/.kube` | Data to send to the endpoint must have a `content` key with content ready to send to the endpoint. Could be from a `configMap`, a `secret` or a `persistentVolumeClaim` as soon as they have a `content` key. |
```

Comment:
the mountPath seems to be different from the one defined in Task

Reply:
corrected into https://github.com/tektoncd/catalog/pull/1002/commits/a7b52b5664e0206f5aba47916c6e87bdd4f9a5ce